packages/preview/splendid-mdpi/0.1.0/template/main.typ

```diff
@@ -0,0 +1,90 @@
+#import "../src/lib.typ" as mdpi
```

Comment:
This should import the package from `@preview`.

Reply:
Resolve in aa690fa6